cli/fixture-features/floating-point-physicals.js

```diff
@@ -0,0 +1,62 @@
+let properties = [
+  ['dimensions ([#133](https://github.com/FloEdelmann/open-fixture-library/issues/133))', 'dimensions'],
+  ['weight', 'weight'],
+  ['power', 'power'],
+  ['color temperature', 'bulb', 'colorTemperature'],
+  ['lumens', 'bulb', 'lumens'],
+  ['lens degrees', 'lens', 'degreesMinMax'],
+  ['pan/tilt max', 'focus', ['panMax', 'tiltMax']]
+];
+
+module.exports = [];
+for (const i in properties) {
+  const property = properties[i];
+  module.exports.push({
+    name: `Floating point ${property[0]}`,
+    description: 'In fixture physical or in a mode\'s physical override.',
+    order: 94-i,
+    hasFeature: function(fixture, fineChannels) {
+      let propertyPath = property.slice(1, property.length);
+      if (inPhysical(fixture.physical, propertyPath)) {
+        return true;
+      }
+      for (const mode of fixture.modes) {
+        if ('physical' in mode && inPhysical(mode.physical, propertyPath)) {
+          return true;
+        }
+      }
+      return false;
+    }
+  });
+}
+
+function inPhysical(physical, propertyPath) {
```

Comment:
This function is a bit complicated to understand. Maybe you can simplify it or at least add some comments.